src/Symfony/Bundle/FrameworkBundle/Resources/config/encryption.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Component\Encryption\EncryptionInterface;
+use Symfony\Component\Encryption\Sodium\SodiumEncryption;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+        ->set('security.encryption.sodium', SodiumEncryption::class)
```

Comment:
```suggestion
        ->set('encryption.sodium', SodiumEncryption::class)
```

I'd skip the `security` prefix because this is not a security sub-component.

Shall we add an alias `encryption` as well, to allow explicit wiring of the service?

Reply:
Are we doing aliases like that? 
Isn't the alias for `EncryptionInterface::class` enough?

Reply:
We usually do, if I'm not mistaken. I find these aliases still helpful for scenarios where I don't use autowiring.